scripts/sipify.py

```diff
@@ -1130,9 +1130,14 @@ def fix_annotations(line):
         r'SIP_PYNAME\(\s*(\w+)\s*\)': r'/PyName=\1/',
         r'SIP_TYPEHINT\(\s*([\w\.\s,\[\]]+?)\s*\)': r'/TypeHint="\1"/',
         r'SIP_VIRTUALERRORHANDLER\(\s*(\w+)\s*\)': r'/VirtualErrorHandler=\1/',
-        r'SIP_THROW\(\s*([\w\s,]+?)\s*\)': r'throw( \1 )',
     }
 
+    if not CONTEXT.is_qt6:
```

Comment:
We don't have sip version available here, do we?
This is also emitting warnings for (py)qt5.

Reply:
We can't make it conditional on sip version as these files are static for qt5 / qt6 builds, and we still support building qt5 with sip4. (And I use sip 4 wherever I can, because it takes a fraction of the time to compile vs 6!)